package.json

```diff
@@ -56,8 +57,8 @@
         "prepare": "husky install && git config feature.manyFiles true",
         "artifacts": "napi artifacts --dist scripts/npm",
         "prepublishOnly": "tsc -d && napi prepublish -p scripts/npm --tagstyle npm",
-        "build": "tsc -d && napi build --platform --release --cargo-name node --cargo-flags=\"-p node\"",
-        "build:dev": "tsc -d && napi build --platform --cargo-name node --cargo-flags=\"-p node\"",
+        "build": "napi build --platform --cargo-name node --js ./node-swc/src/binding.js --dts ./node-swc/src/binding.d.ts -p node --release && tsc -d",
```

Comment:
@kdy1 oh sorry, the `tsc -d` should be before the `napi build`

Reply:
Did it, thank you!